server/rpc_server.go

```diff
@@ -255,6 +255,13 @@ func MonitorRPCRequest() func(ctx context.Context, methodName string, err error)
 	start := time.Now()
 	return func(ctx context.Context, methodName string, err error) {
 		m := rpcEndpointMetrics["rpc."+methodName]
+		if m==nil{
```

Comment:
Please format your code with `gofmt` before committing. I recommend using vim-go or something similar to `gofmt` or `goimports` on each file save. This line should be spaced like this  `if m == nil {`